### ISSUE TITLE
Screen reader access to file explorer

### ISSUE THREAD

**Issue by hturner** [r-wasm/webr#271]
Comment from @ajrgodfrey

> Folders could not be opened with a screen reader. Once a folder was open (with assistance from a sighted person), the screen reader could tab through items inside.
>
> Can create a new file, but then cannot find/open it with a screen reader

I tested a bit further: if a user selects a folder and creates a new file, it will be put in that folder. If the user arrows up/down to a different folder and creates a new file, it will still be put in the last selected folder, i.e. the one that was clicked on not arrowed to - this is a bit confusing and may relate to the screen reader accessibility.

**Comment by georgestagg** [comment]
Thanks, @hturner and @ajrgodfrey for all this great feedback! I was hoping to get time to ask to look at this in person at the R sprint but, alas, I ended up working on other things.

This one is particularly bad, but hopefully it can be fixed since we're using [react-accessible-treeview](https://www.npmjs.com/package/react-accessible-treeview) for the file explorer component.

**Comment by georgestagg** [comment]
There were two related issues here:

1) Selecting a directory using keyboard navigation did not then open that directory. This should now be fixed once commit 7df7964 goes live.

2) It was not clear to sighted users the difference between focusing/hovering on a directory and actually selecting it. In both cases, the visual effect was the same. To keyboard navigate to a different directory, one must keyboard up/down and then press Enter or Space to select it. When commit 7df7964 goes live the visual effects will be different for selection and focus, and so hopefully clearer to sighted users what is happening with the keyboard navigation.

I'm fairly confident navigation with the screen reader should work better now, so I will close the issue. But feel free to let me know with a reply if there are still issues and I will reopen.
